docs/tutorials/eventhandler.rst

```diff
@@ -0,0 +1,75 @@
+###############
+Event Handlers
+###############
+
+For the following let us assume that a Model object is available, which is created as follows:
+
+.. code-block:: python
+
+  from pyscipopt import Model
+
+  scip = Model()
+
+.. contents:: Contents
+
+SCIP Events
+===========
+SCIP provides a number of events that can be used to interact with the solver. These events would describe a change in the model state before or during the solving process; For example a variable/constraint were added or a new best solution is found.
+The enum :code:`pyscipopt.SCIP_EVENTTYPE` provides a list of all available events.
+
+
+Adding Event Handlers
+==============
```

Comment:
I'm never sure on this, but I think you should have at least as many "=" characters as there are in the section titles